### DISCUSSION TITLE
needed service account?

### DISCUSSION THREAD

**Discussion by dreckguy** [QualiTorque/qualitorque.github.io#49] [Documentation requests] [answered]
According to [The Blueprint yaml Doc](https://docs.qtorque.io/blueprint-designer-guide/blueprints) service account must be included but actually the auto-generated blueprints work well without it.
Should we change the docs?
![needed-sa](https://user-images.githubusercontent.com/15980885/175811919-e14a9b48-787b-4493-860a-c329479842b1.png)

**Comment by Shnitselon** [comment]
@shirabendor-quali please advise.

**Comment by shirabendor-quali** [comment, marked as answer]
@dreckguy @Shnitselon I think the reason it works is because you configured the entire cluster to have the correct permissions, Can you double check?
This is an option we also added lately to the docs, but probably better to keep the SA for now as it's more secure/recommended.

**Reply by dreckguy** [reply]
Yes, it used the EKS Node permissions which was enough but the auto-generated blueprints don't refer any.
